### DISCUSSION TITLE
v8 - FullWidth table, depending on headers

### DISCUSSION THREAD

**Discussion by JosepSalvat** [TanStack/table#3982] [Q&A]
Hi, is there a prop or something so we can do the react-table fullWidth and responsive somehow?

Thanks!

**Comment by tannerlinsley** [comment]
V8 is headless, so that’s up to you to implement in your own markup

**Reply by talatkuyuk** [reply]
If the table is fullwidth; `columnSizing` behavior should reflect the pixel changes into adjacent column. 

**While one column is shrinking, the adjacent column has to enlarge to keep fullwidth functionality keeping these sizes in the table columnSizing state.** Additionally the table width should remain the same size `in the table columnSizing state`. 

Is it possible in the react-table?

**Reply by tannerlinsley** [reply]
The column sizing for full width tables depends on if you are using table elements or divs, but regardless of that it should flex relative to Its siblings

**Reply by talatkuyuk** [reply]
In any way; If I solve it via html and css; the react-table `columnSizing state does not change`; and it effects the resizing behavior as reflecting delay, glitch and `unwanted ux behavior`, since the default column size is `150px` and react-table doesn't know `the real width of the column`.

I tried to solve it **non-fullwidth table** https://github.com/TanStack/table/discussions/3947#discussioncomment-2845135. But I think, it is more difficult to do the same with fullWidth tables.
